Makefile

```diff
@@ -213,3 +213,10 @@ hel%: ## help: Show this help message.
 	@echo ""
 	@echo "targets:"
 	@grep -Eh '^.+:\ ##\ .+' ${MAKEFILE_LIST} | cut -d ' ' -f '3-' | column -t -s ':'
+
+deploy-by-day:
```

Comment:
So this is going to be used in the CircleCI builds rather than `deploy` right?

If so, mind tacking on a comment to that effect.